src/main/steps/edge-case/dashboard/template.njk

```diff
@@ -1,5 +1,6 @@
 {% extends "common/template.njk" %}
 
 {% block page_content %}
-<a href="/respondent/task-list" class="govuk-link">Respondent</a>
```

Comment:
@AbdullahShahHMCTS Same comment applies here as well.
We should not be removing respondent link, can we please put it back.